.agent/rules/audius-inspect-details.md

```
@@ -0,0 +1,45 @@
+---
+trigger: model_decision
+description: Inspects the details of the files for multiple audius tracks
+---
+
+Inspects the details of the files for multiple tracks
+GET
+https://api.audius.co/v1/tracks/inspect
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)